TERF:16: update JVM Op Pack README

GabeStah opened the pull request:
Added section explaining Shoreline, Op Packs, and available JVM Op
Packs.

Commit:
TERF:16: update JVM Op Pack README

Added section explaining Shoreline, Op Packs, and available JVM Op
Packs.

timrc-shore commented on README.md:
```diff
@@ -1,14 +1,36 @@
+# Shoreline Terraform JVM Op Packs
 
-This repository is a collection of Shoreline Op Pack Terraform Modules for Java JVM remediation and debugging.
+This repository contains a collection of [Shoreline Op Packs](#what-are-shoreline-op-packs) for Java Virtual Machine (JVM) remediation and debugging.
```
```suggestion
This [Shoreline Op Pack](#what-are-shoreline-op-packs) contains a collection of modules for Java Virtual Machine (JVM) debugging, alerting, and automatic remediation.
```

timrc-shore commented on README.md:
```diff
@@ -1,14 +1,36 @@
-| Module                                                                                                             | Examples                                                                                                            | Description                                                                                       |
-| ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| [JVM Trace]( https://registry.terraform.io/modules/terraform-shoreline-modules/jvm-op-pack/shoreline/latest/submodules/jvm-trace) | [tomcat-to-S3]( https://registry.terraform.io/modules/terraform-shoreline-modules/jvm-op-pack/shoreline/latest/examples/jvm-trace) | Remotely pushes JVM debug data when nodes, pods, or containers exceed a defined memory threshold. |
+The Shoreline platform provides real-time monitoring and incident automation for cloud operations. Use Shoreline to debug and automate repairs across your entire fleet in seconds with just a few lines of code.
```
```suggestion
The Shoreline platform provides real-time monitoring, alerting, and incident automation for cloud operations. Use Shoreline to detect, debug, and automate repairs across your entire fleet in seconds with just a few lines of code.
```

timrc-shore commented on README.md:
```diff
@@ -1,14 +1,36 @@
+
+### What are Shoreline Op Packs?
+
+Shoreline Op Packs are open-source collections of Terraform configurations and supporting scripts that use the [Shoreline Terraform Provider](https://registry.terraform.io/providers/shorelinesoftware/shoreline/latest/docs) to create turnkey incident automations for common operational issues. Each Op Pack comes with smart defaults and works out of the box with minimal setup, while also providing y ...
```
```suggestion
Shoreline Op Packs are open-source collections of Terraform configurations and supporting scripts that use the [Shoreline Terraform Provider](https://registry.terraform.io/providers/shorelinesoftware/shoreline/latest/docs) and the [Shoreline Platform](https://docs.shoreline.io) to create turnkey incident automations for common operational issues. Each Op Pack comes with smart defaults and works out of the box with minimal setup, while also providing you and your team with the flexibility to customize, automate, codify, and commit your own Op Pack configurations.
```

timrc-shore reviewed (changes requested):
Sent some offline comments as well.

GabeStah committed:
Update README.md

Co-authored-by: timrc-shore <67916128+timrc-shore@users.noreply.github.com>

GabeStah committed:
Update README.md

Co-authored-by: timrc-shore <67916128+timrc-shore@users.noreply.github.com>

GabeStah committed:
Update README.md

Co-authored-by: timrc-shore <67916128+timrc-shore@users.noreply.github.com>

Commit:
TERF-16: updated root README

- Adjusted "About Shoreline" section to avoid the implication of
  backend-only capabilities.
- Added small paragraph highlighting Notebooks feature.

per @timrc-shore feedback
